synapse/cortex.py

```diff
@@ -1783,12 +1871,6 @@ def getView(self, iden=None):
         for iden, node in node:
             await self._layrFromNode(node)
 
-        self._migrOrigLayer()
-
-        if self.layers.get(self.iden) is None:
-            # we have no layers.  initialize the default layer.
-            await self.addLayer(iden=self.iden)
-
     def _migrOrigLayer(self):
```

Comment:
we can remove migrOrigLayer now...

Reply:
Discussed. Added comment to remove in 0.3.0